src/Tizen.NUI/src/internal/WebView.cs

```diff
@@ -645,7 +645,7 @@ public ImageView Favicon
             get
             {
                 global::System.IntPtr imageView = Interop.WebView.WebView_GetFavicon(swigCPtr);
-                if (NDalicPINVOKE.SWIGPendingException.Pending)
+                if (imageView == null || imageView == IntPtr.Zero)
```

Comment:
I think comparing IntPtr to null is just same as comparing to IntPtr.Zero.

Reply:
Done.